commands/users.js

```diff
@@ -44,15 +42,15 @@ const handleCommandAsync = async () => {
         });
         console.log(`User with email ${user.email} successfully created`);
       } catch (err) {
-        console.error(err);
+        console.error(err.message);
       } finally {
         exitProgram();
       }
       break;
     }
     case COMMANDS.changePassword: {
       if (args._.length !== 3) {
-        console.error(syntax);
+        console.error(`Wrong syntax: ${COMMANDS.create} password`);
         break;
       }
 
```

Comment:
(opt) it doesn't appear here, but I would put newPassword or something meaningful when you get the new password entry from args:

`const email = args._[1];
      const newPassword = args._[2];`

Reply:
Why? I don't think I follow...